src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateProviderHelper.cs

```diff
@@ -182,18 +182,18 @@ internal ActorStateProviderHelper(IActorStateProviderInternal owner)
             var actorIdList = new List<ActorId>();
             var actorQueryResult = new PagedResult<ActorId>();
 
-            // KVS enumerates its entries in alphabetical order.
+            // Enumerates its entries in alphabetical order.
             var enumerator = getEnumeratorFunc();
 
-            // Move the enumerator to point to first entry
+            // Moves the enumerator to point to first entry.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.